src/SmartComponents/PatchSetWizard/WizardAssets.js

```diff
@@ -163,11 +164,11 @@ export const validatorMapper = {
     },
     'validate-date': () => dateValidator,
     'validate-name': () => (name, formValues) => {
-        if (formValues.takenBaselineNamesLoading || formValues.templateDetailLoading) {
+        if (formValues.areTakenTemplateNamesLoading || formValues.templateDetailLoading) {
             return intl.formatMessage(messages.templateWizardValidateLoading);
         }
 
-        if (formValues.previousName !== name && formValues.takenBaselineNames.includes(name)) {
+        if (formValues.previousName !== name && formValues.takenTemplateNames.includes(name)) {
```

Comment:
When the batched requests for name validation haven't finished, I get this error for every letter in the Name field
![image](https://github.com/RedHatInsights/patchman-ui/assets/20592948/19b5ab96-d737-4bec-8f97-106d570dfa3a)